company/src/main/java/WorkWithStuff.java

```diff
@@ -0,0 +1,130 @@
+import com.github.javafaker.Faker;
+import emploee.*;
+
+
+import java.util.*;
+
+public class WorkWithStuff {
+    public List getStuff() {
+        Stuff stuff[] = Stuff.values();
+        List<Emploee> employees = new LinkedList();
+        Random number = new Random();
+        Faker faker = new Faker();
+        Emploee emploee;
+        for (int i = 0; i < 5; i++) {
+            switch (number.nextInt( 4 - 0 + 1 ) + 0) {
+                case 0:
+                    emploee = new Counter( faker.name().fullName(), 900 );
+                    if(emploee.isHire())
+                        employees.add( emploee );
+                    break;
+                case 1:
+                    emploee =new QA( faker.name().fullName(), 1000 );
+                    if(emploee.isHire())
+                        employees.add( emploee );
+                    break;
+                case 2:
+                    emploee = new Director( faker.name().fullName(), 2000, true );
+
+                        employees.add( emploee );
+                    break;
+                case 3:
+                    emploee  = new Developer( faker.name().fullName(), 1300 );
+                    if(emploee.isHire())
+                        employees.add( emploee );
+                    break;
+                case 4:
+                    emploee =  new Mananger( faker.name().fullName(), 1000 );
+                    if(emploee.isHire())
+                    employees.add( emploee );
+                    break;
+            }
+        }
+        return employees;
+    }
+
+    public List filterBySalary(List stuff, double salary) {
+       List <Emploee>employees = stuff;
+       Emploee emploee;
+        for (int i = 0; i < employees.size() ; i++) {
+            emploee = (Emploee) employees.get( i );
```

Comment:
casting?

Reply:
emploee = (Emploee) employees.get( i );  It means I must write as
 emploee =  employees.get( i );

Reply:
sure